src/components/list-bookmark-button/list-bookmark-button.jsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import PropTypes from "prop-types";
+import currentOfferProp from "../../store/data-props/currentOffers.prop";
+import CitiesBookmarkButton from "../cities-bookmark-button/cities-bookmark-button";
+import PlacesBookmarkButton from "../places-bookmark-button/places-bookmark-button";
+import {CITIES_SUBCLASS, PLACES_SUBCLASS} from "../../const";
+
+const ListBookmarkButton = (
+    {className,
+      offer,
+      authorizationStatus,
+      changeFavoriteAction,
+      refreshOfferList,
+      redirectToRoute,
+      currentOfferId,
+      refreshNearOffersList,
+    }
+) => {
```

Comment:
обычно пары скобок идут вместе
 ```suggestion
const ListBookmarkButton = ({
      className,
      offer,
      authorizationStatus,
      changeFavoriteAction,
      refreshOfferList,
      redirectToRoute,
      currentOfferId,
      refreshNearOffersList,
}) => {
```